lib/addons/neuron/index.ts

```diff
@@ -36,4 +36,19 @@ export class NeuronPluginAddOn implements ClusterAddOn {
 
         return Promise.resolve(pluginStatement);
     }
-}
\ No newline at end of file
+}
+
+export class NeuronMonitorAddOn implements ClusterAddOn {
+    deploy(clusterInfo: ClusterInfo): void {
+        const cluster = clusterInfo.cluster;
+
+        const neuronMonitorDoc = readYamlDocument(__dirname + '/neuron-monitor.yaml');
```

Comment:
Could we also load the `neuron-monitor.yaml` from an external source like Plugin and RBAC?  Im worried about the operational maintenance of YAML. Im sure you have take this YAML from some doc/source repo. Can we render this externally?

Reply:
This has been built, not taken from a source